src/scrabble.js

```diff
@@ -1,5 +1,137 @@
-function scrabble() {
+function isAlpha(str) {
+  return (
+    /^[A-Za-z]*$/.test(str) ||
```

Comment:
Nice use of regex. Can you explain what the **/^[A-Za-z]*$/.test(str)** is meant to do? Thanks.

Reply:
In essence, it checks from a range of A-Z characters and a-z also for case sensitive edge cases